easyaudit/signals/auth_signals.py

```diff
@@ -14,12 +18,14 @@
 def user_logged_in(sender, request, user, **kwargs):
     try:
         with transaction.atomic(using=DATABASE_ALIAS):
-            login_event = audit_logger.login({
-                'login_type': LoginEvent.LOGIN,
-                'username': getattr(user, user.USERNAME_FIELD),
-                'user_id': getattr(user, 'id', None),
-                'remote_ip': request.META[REMOTE_ADDR_HEADER]
-            })
+            audit_logger.login(
+                {
+                    "login_type": LoginEvent.LOGIN,
+                    "username": getattr(user, user.USERNAME_FIELD),
+                    "user_id": getattr(user, "id", None),
+                    "remote_ip": request.META.get(REMOTE_ADDR_HEADER) or "",
```

Comment:
I was hitting issues with this when testing. This resolved #263

Reply:
Moved into its own PR: #272